### DISCUSSION TITLE
Online and Online Meeting are Displayed

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#1020] [Q&A] [answered]
I just noticed that the words "online" and "online meeting" appear in the list of types if it's an online meeting. I don't think this should appear twice.

![Meeting Info](https://user-images.githubusercontent.com/62247421/209981316-bcf66f35-f12c-4ab3-9faa-9a3d8d117b4d.png)

**Comment by joshreisner** [comment, marked as answer]
oops thanks should be fixed now, if you refresh

![Screenshot 2022-12-29 at 9 26 17 AM](https://user-images.githubusercontent.com/1551689/209988206-b31be76c-fdee-4810-be9c-18b865c893a3.png)

**Reply by Grateful-Viv** [reply]
I don't see the change yet. Will this be in a new release or this a cache issue? I have purged cache several times.

Also, if you recall I was having trouble with TSML Legacy showing the wrong times or dates. I purposely restricted meetings from being cached. Is this no longer a need with TSML UI?

**Reply by joshreisner** [reply]
this was a fix in TSML UI so the update is "over the air" - you should see it soon! try switching to a new browser or going to incognito

**Reply by Grateful-Viv** [reply]
yes, I see it now. thank you. 

Would you answer me about the cache question since I'm using TSML UI?

**Reply by joshreisner** [reply]
great

i think you should be able to start caching the meetings page again. TSML UI uses javascript to know what the current time is, and should not be affected
